cli/helptext_test.go

```diff
@@ -48,3 +49,47 @@ func TestSynopsisGenerator(t *testing.T) {
 		t.Fatal("Synopsis should contain options finalizer")
 	}
 }
+
+func TestShortHelp(t *testing.T) {
```

Comment:
Could you also add a test that checks on subcommands? The two things I'd like to watch for are:
1) I'd like some signal that these tests are still useful even if someone changes "For more information about each command" to be rephrased slightly. This could also be aided by checking a constant.
2) Making sure that subcommands are printed